app/models/album.rb

```diff
@@ -0,0 +1,3 @@
+class Album < ActiveRecord::Base
+  validates :name, presence: true, uniqueness: true
+end
```

Comment:
This would be where you could add in those scopes mentioned in earlier comment.

Example:
`scope :best, -> (total) { order('upvotes DESC').limit(total)}`